### PR TITLE
Move Internal LB IP to a well known offset

### PR DESCRIPTION
**Pull request by raveeram**
- ILB IP will now be at (firstMasterIP + 10). Default: 10.240.255.15
- This will make it easy to build illustration diagrams/examples for kubernetes.
- Also, having a consistent LB IP address, makes troubleshooting customer incidents easier

### CODE REVIEW

File: parts/kubernetesmastervars.t [hunk 1]
```diff
@@ -75,7 +75,8 @@
     "masterInternalLbID": "[resourceId('Microsoft.Network/loadBalancers',variables('masterInternalLbName'))]",
     "masterInternalLbIPConfigName": "[concat(variables('orchestratorName'), '-master-internal-lbFrontEnd-', variables('nameSuffix'))]",
     "masterInternalLbIPConfigID": "[concat(variables('masterInternalLbID'),'/frontendIPConfigurations/', variables('masterInternalLbIPConfigName'))]",
-    "masterInternalLbIp": "[concat(variables('masterFirstAddrPrefix'), add(variables('masterCount'), int(variables('masterFirstAddrOctet4'))))]",
+    "masterInternalLbIPOffset": 10,
```
Comment (colemickens) [inline parts/kubernetesmastervars.t]: You're not using the constant you defined.

Comment (anhowe) [inline parts/kubernetesmastervars.t]: Update the commit message with the most important reason: "having a consistent LB IP address, makes troubleshooting customer incidents easier".

Comment (raveeram) [inline parts/kubernetesmastervars.t]: Whoops, good catch. Didn't update this PR.

File: pkg/acsengine/const.go [hunk 2]
```diff
@@ -23,6 +23,9 @@ const (
 	// DefaultKubectlVersion is the version used for kubectl
 	// The latest stable version can be found here: https://storage.googleapis.com/kubernetes-release/release/stable.txt
 	DefaultKubectlVersion = "v1.5.1"
+	// DefaultInternalLbStaticIPOffset specifies the offset of the internal LoadBalancer's IP
+	// address relative to the first consecutive Kubernetes static IP
+	DefaultInternalLbStaticIPOffset = 10
```
Comment (anhowe) [inline pkg/acsengine/const.go]: discard this since it is not used

File: parts/kubernetesmastervars.t (hunk 1 above)
Comment (raveeram) [inline parts/kubernetesmastervars.t]: @colemickens it's the diff. should've added it below. See the line below "masterInternalLbIpOffset"

File: pkg/acsengine/const.go (hunk 2 above)
Comment (anhowe) [inline pkg/acsengine/const.go]: better yet, wire it up to the vars

File: parts/kubernetesmastervars.t (hunk 1 above)
Comment (raveeram) [inline parts/kubernetesmastervars.t]: @anhowe added.

**Pull request merged**